### PR TITLE
Add QA jobs for GCI M56

### PR DESCRIPTION
**Pull request by Amey-D**
@wonderfly could you please review?

cc @kubernetes/goog-image FYI

### CODE REVIEW

**Comment by wonderfly** [comment]
:lgtm:

---

Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/test-infra/1300#-:-KYGcYw1iA2gO0ddm7Wo:bnfp4nl)*

**Comment by krzyzacy** [comment]
please add the job itself, to `/jobs`

**Comment by wonderfly** [comment]
> please add the job itself, to /jobs

@krzyzacy Sorry what did you mean by `/jobs`? Can you comment inline with code if possible?

**Comment by krzyzacy** [comment]
@wonderfly all the job scripts live in https://github.com/kubernetes/test-infra/tree/master/jobs, I mean it's not a magic you add a named job entry it will figure out how it works :-)

**Comment by wonderfly** [comment]
> @wonderfly all the job scripts live in https://github.com/kubernetes/test-infra/tree/master/jobs, I mean it's not a magic you add a named job entry it will figure out how it works :-)

This is new to me, and I believe to @Amey-D as well. So now all job configs are moved to their own shell scripts?

**Comment by wonderfly** [comment]
@krzyzacy I am assigning this to you. Please advise on how to add the jobs correctly. Many have changed since I went on vacation. :)

**Comment by krzyzacy** [comment]
@wonderfly sure, yes it is new, we migrated everything using bootstrap.py, and hope later on can get away from jenkins.

@Amey-D you need to create/add files like `ci-kubernetes-e2e-gce-gci-qa-m56.sh` to https://github.com/kubernetes/test-infra/tree/master/jobs, probably just reference https://github.com/kubernetes/test-infra/blob/master/jobs/ci-kubernetes-e2e-gce-gci-qa-m55.sh and update necessary fields.

**Comment by Amey-D** [comment]
I have added the definitions for the new test jobs. PTAL.

File: jobs/ci-kubernetes-e2e-gce-gci-qa-m56.sh [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+# Get golang into our PATH so we can run e2e.go
+export PATH="${PATH}:/usr/local/go/bin"
+
+### Runner
+readonly runner="${testinfra}/jenkins/dockerized-e2e-runner.sh"
+export KUBEKINS_TIMEOUT="50m"
+timeout -k 20m "${KUBEKINS_TIMEOUT}" "${runner}" && rc=$? || rc=$?
+
+### Reporting
```
Comment (krzyzacy) [inline jobs/ci-kubernetes-e2e-gce-gci-qa-m56.sh]: if you ref https://github.com/kubernetes/test-infra/blob/master/jobs/ci-kubernetes-e2e-gce-gci-qa-m55.sh

this section is already refactored, and should be something like 

### Runner
```
readonly runner="${testinfra}/jenkins/dockerized-e2e-runner.sh"
export DOCKER_TIMEOUT="70m"
export KUBEKINS_TIMEOUT="50m"
"${runner}"
```

same as the other ones

**Comment by krzyzacy** [comment]
looks fine, do you have those new GCE projects ready?

**Comment by krzyzacy** [comment]
/lgtm

let me know when you are ready to kick off the jobs and I'll merge them

**Comment by Amey-D** [comment]
The GCE projects are ready.  Feel free to merge the PR.

**Comment by krzyzacy** [comment]
:lgtm:

---

Review status: 1 of 5 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/test-infra/1300#-:-KYLs3Ix3LzpJenC_JCt:bnfp4nl)*

**Pull request merged**